src/aggregate.d

```diff
@@ -215,78 +208,136 @@ public:
             sd.semanticTypeInfoMembers();
     }
 
-    abstract void finalizeSize();
-
-    override final uint size(Loc loc)
+    /***************************************
+     * Find all instance fields, then push them into `fields`.
```

Comment:
The new fields introduced by senamtic analysis of tuples are iterated properly, because we use `for` loop to iterate `members`.

If, more fields are added after the `detemrinieField` process, it will be rejected by the error in `VarDeclaration.semantic`:
`Error: cannot be further field because it will change the determined`